test/Dialect/Torch/decompose-complex-ops.mlir

```diff
@@ -1024,3 +1024,32 @@ func.func @torch.aten.baddbmm(%arg0: !torch.vtensor<[?,?,?],f32>, %arg1: !torch.
   %0 = torch.aten.baddbmm %arg0, %arg1, %arg2, %int1, %int1 : !torch.vtensor<[?,?,?],f32>, !torch.vtensor<[?,?,?],f32>, !torch.vtensor<[?,?,?],f32>, !torch.int , !torch.int -> !torch.vtensor<[?,?,?],f32>
   return %0 : !torch.vtensor<[?,?,?],f32>
 }
+
+// CHECK-LABEL:   func.func @torch.aten.repeat(
+// CHECK-SAME:                 %[[ARG0:.*]] !torch.vtensor<[3,1],f32>) -> !torch.vtensor<[2,12,5],f32> {
```

Comment:
Does it also work when the inputs are dynamic?

Reply:
I have changed to dynamic inputs